core/src/test/java/org/jdbi/v3/core/transaction/TestLocalTransactionHandler.java

```diff
@@ -53,4 +54,19 @@ public void testRollbackThrow() throws Exception {
             assertThat(e.getSuppressed()[0]).isSameAs(inner);
         }
     }
+
+    @Test
+    public void testThrowError() throws Exception {
+        Error error = new Error("Transaction throws!");
+
+        Mockito.when(c.getAutoCommit()).thenReturn(true);
+        Mockito.when(h.getConnection()).thenReturn(c);
+
+        // Verify that we don't get a ClassCastException when an Error is thrown within a transaction.
```

Comment:
Well if you're going to write a comment to explain the test case, best also put it in the appropriate place, a `describedAs` call. But I'm not sure a comment is the way to go. An assertion like `isNotInstanceOf(ClassCastException.class)` would achieve the same but in a tangible way.